README.md

```diff
@@ -35,14 +32,12 @@ This repository also hosts the documentation for the `Glean SDK`:
 * [Documentation of the Rust crate][rustdoc].
 * [Documentation of the Kotlin library][ktdoc].
 
-For an overview of Glean, see the [section in the Firefox data docs](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
-
 ## Contact
 
 To contact us you can:
 
 * Find us on the Mozilla Slack in *#glean*, on [Mozilla IRC][mozirc] in *#telemetry*.
-* To report issues or request changes, file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK][newbugzilla].
+* To report issues or request changes, file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data%20Platform%20and%20Tools&component=Glean%3A%20SDK).
```

Comment:
Mh, let's leave the old link there (it's referencing a link at the end of this file) for now: we need to decide how to handle the two triages and how to file bugs for the two different products.